docs/legacy-ie.md

```diff
@@ -4,7 +4,36 @@ Unless stated otherwise the approach to browser support still follows the GDS pr
 
 We want to continue providing a functional experience for user of legacy browsers, but in a way that reduces complexity in this codebase.
 
-## Separate stylesheet
+## Generate your own legacy IE stylesheet
+If you're compiling your own stylesheets from GOV.UK Frontend component partials, you can generate a separate stylesheet for legacy browsers using a build tool (like [Gulp](https://gulpjs.com/))
+
+In our partials we use IE mixins that generate code for versions of IE:
+
+```
+@include ie-lte(8) {
+  border-bottom: $button-shadow-size solid $govuk-button-colour-darken-15;
+}
+```
+
+as well [SassMQ functionality](https://sass-mq.github.io/sass-mq/#undefined-variable-mq-responsive) that flattens media queries.
```

Comment:
Not sure this is as clear as it could be. Something like 'You need to set responsive to false, otherwise IE8 will get mobile styles, because it doesn't support media queries'?